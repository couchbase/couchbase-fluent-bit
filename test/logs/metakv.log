[metakv:debug,2021-03-09T17:32:01.613Z,ns_1@cb.local:simple_store_xdcr_ckpt_data<0.382.0>:simple_store:init:114]Creating Table: xdcr_ckpt_data
[metakv:debug,2021-03-09T17:32:02.113Z,ns_1@cb.local:<0.410.0>:menelaus_metakv:handle_iterate:133]Starting iteration of /remoteCluster/. Continuous = false
[metakv:debug,2021-03-09T17:32:14.993Z,ns_1@cb-example-0000.cb-example.default.svc:<0.421.0>:menelaus_metakv:handle_iterate:133]Starting iteration of /. Continuous = true
[metakv:debug,2021-03-09T17:32:14.994Z,ns_1@cb-example-0000.cb-example.default.svc:<0.421.0>:menelaus_metakv:write_chunk:161]Sent /indexing/settings/config rev: g2wAAAABaAJtAAAAIDZmYzA4YzU4MThmZmE1MmRjNmM2MjRlZGEzMWJkZGMxaAJhA24FAB4tvNkOag== sensitive: false
[metakv:debug,2021-03-09T17:32:15.256Z,ns_1@cb-example-0000.cb-example.default.svc:<0.1221.0>:menelaus_metakv:handle_mutate:90]Updated <<"/query/dictionary_cache/counter">>. Elapsed time:0 ms.
