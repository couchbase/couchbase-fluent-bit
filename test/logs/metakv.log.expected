/fluent-bit/test/logs/metakv.log.actual: [1615311121.613000000, {"file_path":"/fluent-bit/test/logs","file":"metakv.log","filename":"/fluent-bit/test/logs/metakv.log","logger":"metakv","timestamp":"2021-03-09T17:32:01.613Z","message":"ns_1@cb.local:simple_store_xdcr_ckpt_data<0.382.0>:simple_store:init:114]Creating Table: xdcr_ckpt_data","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/metakv.log.actual: [1615311122.113000000, {"file_path":"/fluent-bit/test/logs","file":"metakv.log","filename":"/fluent-bit/test/logs/metakv.log","logger":"metakv","timestamp":"2021-03-09T17:32:02.113Z","message":"ns_1@cb.local:<0.410.0>:menelaus_metakv:handle_iterate:133]Starting iteration of /remoteCluster/. Continuous = false","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/metakv.log.actual: [1615311134.993000000, {"file_path":"/fluent-bit/test/logs","file":"metakv.log","filename":"/fluent-bit/test/logs/metakv.log","logger":"metakv","timestamp":"2021-03-09T17:32:14.993Z","message":"ns_1@cb-example-0000.cb-example.default.svc:<0.421.0>:menelaus_metakv:handle_iterate:133]Starting iteration of /. Continuous = true","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/metakv.log.actual: [1615311134.994000000, {"file_path":"/fluent-bit/test/logs","file":"metakv.log","filename":"/fluent-bit/test/logs/metakv.log","logger":"metakv","timestamp":"2021-03-09T17:32:14.994Z","message":"ns_1@cb-example-0000.cb-example.default.svc:<0.421.0>:menelaus_metakv:write_chunk:161]Sent /indexing/settings/config rev: g2wAAAABaAJtAAAAIDZmYzA4YzU4MThmZmE1MmRjNmM2MjRlZGEzMWJkZGMxaAJhA24FAB4tvNkOag== sensitive: false","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/metakv.log.actual: [1615311135.256000000, {"file_path":"/fluent-bit/test/logs","file":"metakv.log","filename":"/fluent-bit/test/logs/metakv.log","logger":"metakv","timestamp":"2021-03-09T17:32:15.256Z","message":"ns_1@cb-example-0000.cb-example.default.svc:<0.1221.0>:menelaus_metakv:handle_mutate:90]Updated <<\"/query/dictionary_cache/counter\">>. Elapsed time:0 ms.","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
